src/models/meer10/README.md

```diff
@@ -0,0 +1,53 @@
+# Meerkat (meer10)
+
+- [Parts & Repairs](./repairs.md)
+
+![Meerkat (short)](./img/meer10.webp)
+
+The System76 Meerkat is a desktop with the following specifications:
+
+- CPU
+    - Supports Intel Core Ultra 2nd generation (Arrow Lake) and Core 14th generation (Raptor Lake) CPUs
+        - [Core Ultra 7 255H](https://www.intel.com/content/www/us/en/products/sku/241751/intel-core-ultra-7-processor-255h-24m-cache-up-to-5-10-ghz/specifications.html)
+        - [Core Ultra 5 225H](https://www.intel.com/content/www/us/en/products/sku/241749/intel-core-ultra-5-processor-225h-18m-cache-up-to-4-90-ghz/specifications.html)
+        - [Core 3 100U](https://www.intel.com/content/www/us/en/products/sku/236776/intel-core-3-processor-100u-10m-cache-up-to-4-70-ghz/specifications.html)
+- Graphics
+    - GPU: Dependent on CPU
+        - Core Ultra 7 255H: Intel Arc 140T GPU
+            - Requires 2 SO-DIMMs for Arc graphics
+        - Core Ultra 5 225H: Intel Arc 130T GPU
+            - Requires 2 SO-DIMMs for Arc graphics
+        - Core 3 100U: Intel Graphics (integrated)
+    - Video output:
+        - 2x HDMI 2.1
+        - 2x DisplayPort 2.1 over USB-C
+- Memory
+    - Up to 96GB (2x48GB) Dual Channel DDR4 SO-DIMMs @ 5600 MHz
+- Networking
+    - 1x 2.5-Gigabit Ethernet (built-in)
+    - 1x 2.5-Gigabit Ethernet (optional add-on)
+    - 1x WiFi 7
+- Power
+    - Core Ultra 7 and Ultra 5 models:
+        - 120W (19V, 6.32A) DC-in port
+            - Barrel size: 5.5mm (outer), 2.5mm (inner)
+    - Core 3 model:
+        - 90W (19V, 4.74A) DC-in port
+            - Barrel size: 5.5mm (outer), 2.5mm (inner)
+- Sound
+    - 7.1-channel digital audio output over HDMI and USB-C
+- Storage
```

Comment:
The expansion module that enables a second 2280 M.2 drive occupies the 2242 slot, so these cannot use three drives. Either the 2242 (or 2230 with an adapter) and the 2280, or two 2280's with the expansion module.

Reply:
That's a little unclear on https://system76.com/desktops/meerkat/#specs, but I'll make it clear here.

Reply:
How's that?

Reply:
Looks good!